### PR TITLE
Fixes/win32 can resize

### LINKED ISSUE #4941: Bug when CanResize is false
Now I'm making custom windows and I found a problem, when the ```CanResize``` property is false and the method ```Window.BeginResizeDrag(WindowEdge, PointerPressedEventArgs)``` is executed, the window can still be resized, in a nutshell, CanResize does not work; I'd want to know if there is a way to avoid the bug, or if you are thinking of eliminating the error.

### LINKED ISSUE #5526: CanResize="False" and ExtendClientAreaToDecorationsHint="True" do not work together
**Describe the bug**
When setting `CanResize="False"` with `ExtendClientAreaToDecorationsHint="True"` you lose the ability to move the window.

**To Reproduce**
Using the following AXAML will reproduce the issue: https://gist.github.com/Grinch/7bfb1d3d7d30bcd1d2164e49be3d1824

**Expected behavior**
Ability to drag the window by the title bar.

**Desktop (please complete the following information):**

- OS: Windows 10 (Build 21318)
- Version: 0.10.0

### LINKED ISSUE #6942: CanResize="False" with removed system title bar makes the window impossible to move
**Describe the bug**
`CanResize="False"`
with
```
ExtendClientAreaChromeHints="PreferSystemChrome"
ExtendClientAreaTitleBarHeightHint="-1"
ExtendClientAreaToDecorationsHint="True"
```
makes the window impossible to move using the title bar.

**To Reproduce**

1. Set these parameters in your Window settings:
```
CanResize="False"
ExtendClientAreaChromeHints="PreferSystemChrome"
ExtendClientAreaTitleBarHeightHint="-1"
ExtendClientAreaToDecorationsHint="True"
```
2. Try to drag the window using the title bar
3. See it doesn't move

**Expected behavior**
With `CanResize="False"` enabled and the title bar being custom, the user should still be able to move the window around using the title bar.

**Desktop (please complete the following information):**

- OS: Windows
- Version 11 (22000.318)

### PR DESCRIPTION
**Pull request by danwalmsley**
## What is the current behavior?

If you have ExtendToClientDecorations true, and CanResize=false... you cant drag with titlebar on win32.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

## Fixed issues

Fixes #6942
Fixes #5526
Fixes #4941

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019044-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019044-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Windows/Avalonia.Win32/WindowImpl.cs [hunk 1]
```diff
@@ -633,13 +633,16 @@ public void BeginMoveDrag(PointerPressedEventArgs e)
 
         public void BeginResizeDrag(WindowEdge edge, PointerPressedEventArgs e)
         {
+            if (_windowProperties.IsResizable)
+            {
 #if USE_MANAGED_DRAG
             _managedDrag.BeginResizeDrag(edge, ScreenToClient(MouseDevice.Position.ToPoint(_scaling)));
```
Comment (robloo) [inline src/Windows/Avalonia.Win32/WindowImpl.cs:639]: nit: Indentation

Comment (danwalmsley) [reply src/Windows/Avalonia.Win32/WindowImpl.cs]: fixed

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019062-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by Takoooooo — APPROVED** [review]
LGTM

**Pull request merged**